spec/middleware/exception_hunter/request_hunter_spec.rb

```diff
@@ -69,6 +70,40 @@ module ExceptionHunter
           error = Error.last
           expect(error.environment_data).not_to include(not_tracked_env)
         end
+
+        context 'when the exception was raised by a logged user' do
+          let!(:user) { User.create!(email: 'example@example.com', password: 'password') }
```

Comment:
We don't really need to create a user here, we could just use a mock again to decouple from the dummy app. Also, remember to use `#let` whenever possible since it's lazy.